tarpc/Cargo.toml

```diff
@@ -53,7 +53,7 @@ tarpc-plugins = { path = "../plugins", version = "0.13" }
 thiserror = "1.0"
 tokio = { version = "1", features = ["time"] }
 tokio-util = { version = "0.7.3", features = ["time"] }
-tokio-serde = { optional = true, version = "0.8" }
+tokio-serde = { optional = true, git = "https://github.com/tamird/tokio-serde", branch = "remove-educe", features = ["json", "bincode"] }
```

Comment:
I'd rather depend on tokio-serde even if it pulls in 2 versions of syv. Consider a PR to tokio-serde?